packages/components/src/presentation-components/instance-filter-builder/InstanceFilterConverter.ts

```diff
@@ -224,7 +224,11 @@ function handleStringifiedValues(filter: PresentationInstanceFilterCondition, se
   let selectedValueIndex = 0;
 
   const { displayValues, groupedRawValues } = deserializeDisplayValueGroupArray(serializedDisplayValues, serializedGroupedRawValues);
-  if (displayValues === undefined || groupedRawValues === undefined) {
+  if (
+    displayValues === undefined ||
+    groupedRawValues === undefined ||
+    (operator !== PropertyFilterRuleOperator.IsEqual && operator !== PropertyFilterRuleOperator.IsNotEqual)
```

Comment:
I think operator should be checked in `convertUniqueValuesCondition` function before trying to deserialize values.

Reply:
fixed

Reply:
fixed